client/src/PlayerGraph.js

```diff
@@ -0,0 +1,150 @@
+import d3 from 'd3'
+
+export default class PlayerGraph {
+
+  init (node) {
+    this.margin = {top: 60, right: 30, bottom: 30, left: 30}
+    this.width = node.clientWidth - this.margin.left - this.margin.right
+
+    if (this._isSmallScreen()) {
+      this.margin.bottom = 60
+    }
+
+    this.height = 500 - this.margin.top - this.margin.bottom
+
+    this.x0 = d3.scale.ordinal()
+        .rangeRoundBands([0, this.width], 0.1)
+
+    this.x1 = d3.scale.ordinal()
+
+    this.y = d3.scale.linear()
+        .range([this.height, 0])
+
+    this.klass = d3.scale.ordinal()
+        .range(['playerA', 'playerB'])
+
+    this.xAxis = d3.svg.axis()
+        .scale(this.x0)
+        .orient('bottom')
+
+    this.yAxis = d3.svg.axis()
+        .scale(this.y)
+        .orient('left')
+
+    this.chart = d3.select(node).append('svg')
+        .attr('width', this.width + this.margin.left + this.margin.right)
+        .attr('height', this.height + this.margin.top + this.margin.bottom)
+      .append('g')
+        .attr('transform', 'translate(' + this.margin.left + ',' + this.margin.top + ')')
+  }
+
+  _isSmallScreen () {
+    return this.width < 500
+  }
```

Comment:
the chart is semi responsive. It rotates axis if the screen is small. This also needs a bigger bottom margin then